BSB_lan_defs.h

```diff
@@ -8315,7 +8383,11 @@ PROGMEM_LATE const cmd_t cmdtbl2[]={
 {0x053D0DA3,  CAT_IOTEST,           VT_CLOSEDOPEN,    7841,  STR7841,  sizeof(ENUM_CLOSEDOPEN), ENUM_CLOSEDOPEN, FL_RONLY,     DEV_203_ALL}, // [0] - Ein-/Ausgangstest - Kontaktzustand H1
 {0x053D0DA3,  CAT_IOTEST,           VT_CLOSEDOPEN,    7841,  STR7841,  sizeof(ENUM_CLOSEDOPEN), ENUM_CLOSEDOPEN, FL_RONLY,     DEV_205_ALL}, // [0] - Ein-/Ausgangstest - Kontaktzustand H1
 {0x053D0DA3,  CAT_IOTEST,           VT_CLOSEDOPEN,    7841,  STR7841,  sizeof(ENUM_CLOSEDOPEN), ENUM_CLOSEDOPEN, FL_RONLY,     DEV_211_ALL}, // [0] - Ein-/Ausgangstest - Kontaktzustand H1
-{0x053D171C,  CAT_IOTEST,           VT_UNKNOWN,       7844,  STR7844,  0,                    NULL,         FL_RONLY,     DEV_ALL}, // Eingangssignal H1
+
+// Elco Wärmepumpe RVS61.843E/560 - Signalart H1 [Kein ¦ Geschl' (ooo), Offen (---) ¦ Impulse ¦ Frequenz Hz ¦ Spannung V]
+// "---" == DC 80 0A 10 07 05 3D 17 1C 01 00 00 00 00 BF A0
+{0x053D171C,  CAT_IOTEST,           VT_CUSTOM_ENUM,   7844,  STR7844,  sizeof(ENUM7844),        ENUM7844,        FL_RONLY,     DEV_ALL}, // Eingangssignal H1
```

Comment:
Can you send me a telegram which is not "---" (i.e. disabled)? VT_CUSTOM_ENUM is definitively wrong even though it may work accidentially. VT_CUSTOM_ENUM expects two different variables encoded into one telegram and this is not the case here. I may have to add a new data type if VT_ENUM is really not working here, but let me see a proper (non-disabled) telegram first.

Reply:
This parameter shows type and value of signal on H1 - currently I don't see possibility to change value of this parameter.

Reply:
See below.